python_ta/reporters/templates/stylesheet.css

```diff
@@ -157,6 +157,11 @@ header {
   list-style-type: none;
   padding: 0px 20px;
   white-space: nowrap;
+  color: var(--text-primary);
```

Comment:
These styles shouldn't be added; in #1202, they were added to `.sidebar button`.

As an aside, these style cause the rendering to look pretty bad. You should always make sure to check your changes manually by regenerating the report. It's also a good practice to update the screenshots in the PR description for UI changes.